src/components/Auth/SignUp/SignUp.css

```diff
@@ -33,13 +33,19 @@
   width : 50%;
   float : right;
 }
-.app-body{
+
+.signup-container h1:nth-child(2){
+  margin-top:10px;
+  font-weight:600;
+}
+.app-body-div{
   height: 80%;
   width : 50%;
   float : left;
   background-color : #EEE;
 }
 
+
```

Comment:
Remove extra line  . 
Also resolve the merge conflicts and squash the commits as asked by @rossoskull .